src/monmove.cpp

```diff
@@ -415,7 +416,14 @@ void monster::anger_cub_threatened( monster_plan &mon_plan )
     }
 
     for( monster &tmp : g->all_monsters() ) {
+        bool is_baby = false;
         if( type->baby_monster == tmp.type->id ) {
+            is_baby = true;
+        }
+        if( MonsterGroupManager::IsMonsterInGroup( type->baby_monster_group, tmp.type->id ) ) {
+            is_baby = true;
+        }
+        if( is_baby ) {
```

Comment:
```suggestion
        const bool is_baby = type->baby_monster == tmp.type->id
                             || MonsterGroupManager::IsMonsterInGroup( type->baby_monster_group, tmp.type
        if( is_baby ) {
```

Reply:
should it be simplified further as 
```suggestion
        if( type->baby_monster == tmp.type->id || MonsterGroupManager::IsMonsterInGroup( type->baby_monster_group, tmp.type ) ) {
```
?

Reply:
It can be. It is up to the programmer. It is more readable sometimes, to have a named const bool.